src/common/utils/setup-helpers.js

```diff
@@ -618,6 +618,32 @@ function printSetupMessages(): void {
   setupMessages.forEach(msgObj => getLogger('KalturaPlayer:Setup')[msgObj.level](msgObj.msg));
 }
 
+/**
+ * Prints early setup messages.
+ * @private
+ * @param {Player} player - The player.
+ * @param {string} engine - The player engine name.
+ * @param {string} format - The player engine format.
+ * @returns {PKPlaybackConfigObject} - The playback config.
+ */
+function addEngineToStreamPriority(player: Player, engine: string, format: string): PKPlaybackConfigObject {
```

Comment:
i don't like the idea of adding code for specific engines inside the player itself, since that defeats the idea of engines being pluggable.
is this the only place that this code can be in ?